level.go

```diff
@@ -1,18 +1,44 @@
 package engo
 
+import (
+	"engo.io/gl"
+)
+
 type Level struct {
-	Width      int
-	Height     int
+	width      int
```

Comment:
> It'd be good to have a function to get width/height of the map in pixels?

As long as we don't ever call it "pixels" - because they're only pixels as long as the zoom-level is at 100% :smile: (and there's no scaling/resizing).